README.md

```diff
@@ -68,44 +70,38 @@ DBT must always be run from the *root directory* of the repository. The remainde
 
-#### Build variables:
+#### Build targets:
 
-* `OLED` - narrow the scope to only build targets for OLED or only build targets for 7segment displays. When left out, all display types are built. This variable can be stacked with other variables and applies to `clean` as well.
-    * eg: `./dbt all OLED=1` (build all **OLED** targets)
-    * eg: `./dbt all OLED=0` (build all **7segment** targets)
-* `DEBUG` - narrow the scope to only build targets with debugging support (including debugging symbols and different optimizations) or only release targets. When left out, both debugging and release targets are built. This variable can be stacked with other variables and applies to `clean` as well.
-    * eg: `./dbt all DEBUG=1` (build only **debugging** targets)
-    * eg: `./dbt all DEBUG=0` (build only **release** targets)
+* `all` - build all supported targets (7seg, oled)
+* `oled` - build only for OLED.
+* `7seg` - build only for 7SEG.
+
+#### Build configurations:
+
+* `debug` - build selected target with debugging support (including debugging symbols and different optimizations) 
+* `release` - build selected target with optimizations for release.
 
 #### Build arguments: 
 
-* `-c` or `--clean` - this is the built-in SCons clean option. When used with `./dbt all` it's equivalent to `./dbt clean`.
-* `--e2_orig_prefix` - this turns on support for the prefixes e2 Studio expects based on the original Deluge source tree. All instances of `dbt-build-` will be replaced with `e2-build-` and the output directories will change as well.
-* `--nowo`, `--no-owo`, or `--no-uwu` - DBT features a progress indicator (not always visible) in the form of an ASCII emote face that switches rapidly between OwO, UwU, etc. If this is too noisy in the logs, it can be turned off by adding one of these arguments.
-* `--verbose=[0-3]` - This controls the noise level of DBT and its underlying calls:
-    * `0` - Bare minimum. No notices, hidden warnings, and only critical errors are shown.
-    * `1` - Clean. Hidden warnings, and build steps are described without showing the actual command.
-    * `2` - Commands and warnings are visible.
-    * `3` - Everything is shown where possible.
-* `--spew` - This is shorthand for `--verbose=3` and `--nowo`, each described above.
+* `-S` or `--no-status` - this disables the `ninja` status line while building. Output from compilation commands (such as warnings and errors) will still be printed.
+* `-v` or `--verbose` - this prints greater level of detail to the console (i.e. exactly the compiler/linker commands called)
+* `-c` or `--clean-first` - clean before building 
 
-`clean` - Clean up all supported targets (dbt-build-release-oled, dbt-build-release-7seg, dbt-build-debug-oled, and dbt-build-debug-7seg)
-* Variables described above under `all` also apply to `clean`.
+#### CMake custom arguments
+
+Any additional arguments to CMake may be transparently passed via `dbt build`.
```

Comment:
Document what these are (ENABLE_RTT and FORCE_COLORED_OUTPUT are the 2 I can see immediately) would probably be good.

Reply:
Those aren't arguments, they're configure options, and they can be listed with the typical `cmake -B build -LH` or edited with cmake-gui, and are internal use anyways. Arguments are things listed by `cmake --build build`, and the only relevant one I can see is the number of parallel jobs, which I'll add here.